### DISCUSSION TITLE
sanity-plugin-table migration to V3

### DISCUSSION THREAD

**Discussion by davydog187** [sanity-io/sanity#3858] [Sanity Studio Support]
Hello! I'm looking for guidance on publishing a fork of the popular [Sanity Table Plugin](https://www.sanity.io/plugins/sanity-plugin-table) that I've migrated to Sanity V3.

### Background
Unfortunately, the plugin has [has not been maintained](https://github.com/rdunk/sanity-plugin-table) for some time, although [a fork](https://github.com/MathisBullinger/sanity-plugin-another-table) did spend some time merging in bugfixes. My company makes heavy use of this table, so we plan to take stewardship over the table and add additional features, such as PortableText support. I have created my own fork that grabs these bugfixes, and then begun the migration to Sanity v3.

### Migration Work
* [My fork with the V3 migration](https://github.com/bitfo/sanity-plugin-table)
* [PR migrating to V3](https://github.com/bitfo/sanity-plugin-table/pull/1)

### Future Plans
* Support PortableText more generally in table cells

### Related Discussion
- [V3 enabled table plugins](https://github.com/sanity-io/sanity/discussions/3821)

### Questions
I want to publish a V3 version to npm and get the official Sanity page updated once the migration is complete. My concern is regarding how to publish with minimal confusion to the community

1. How should I handle publishing to npm given that the original already exists under the name [sanity-plugin-table](https://www.npmjs.com/package/sanity-plugin-table)?
2. Should I diverge the plugin name from `table` to something else? I think it would be best to keep the same. (The other fork renamed it to `another-table` which I was not a fan of).
3. How should we handle updating the official Sanity page?

I'm happy to do whatever is in the best interest of the community, but since we are maintaining this fork, I would prefer to keep it under our control.

**Comment by davydog187** [comment]
Also paging @rdunk and @MathisBullinger. Ideally we can find a solution where we get maintainer access to the canonical repo and we can further it there.

**Reply by snorrees** [reply]
[@sanity/table 1.0.0 ](https://github.com/sanity-io/table/releases/tag/v1.0.0)for v3 is now out.
This version is officially under Sanity.io stewardship for v3 moving forward.

Thanks a lot for this work @davydog187 ! All contributors original and otherwise should be credited in the MIT license and readme. 
Let me know you anyone are left out.
